common/views/components/ButtonSolid/ButtonSolid.tsx

```diff
@@ -157,51 +157,25 @@ export const SolidButton = styled(BaseButton).attrs<SolidButtonProps>(
   })
 )<SolidButtonProps>`
   background: ${props =>
-    props.theme.color(
+    props.theme.newColor(
       props?.colors?.background || props.theme.buttonColors.default.background
     )};
   color: ${props =>
-    props.theme.color(
+    props.theme.newColor(
       props?.colors?.text || props.theme.buttonColors.default.text
     )};
   border: 2px solid
     ${props =>
-      props.theme.color(
+      props.theme.newColor(
         props?.colors?.border || props.theme.buttonColors.default.border
       )};
 
   padding: ${({ size }) => getPadding(size)};
 
+  // TODO use a function to lighten the colours on hover
+  // In the meantime, use underline
```

Comment:
Aligning behaviour of buttons until we decide how we want to do hover behaviour. Gareth O was suggesting lightening the colours, we talked about using a function, etc. We'll need to decide how that looks on outline buttons as well.